### DISCUSSION TITLE
Unable to launch Due to Invalid Name Appimage

### DISCUSSION THREAD

**Discussion by thyTwilightGoth** [bengtmartensson/IrScrutinizer#498] [Q&A] [answered]
Hi

I am not entirely sure this is the right place for me to put this issue, However given that I do not have this issue with other appimages I will be putting this issue on this project for the time being. As well as it seems like I might have the only documented case of this error happening, given that the error brought up zero similar search results.

Onto the actual issue, whenever I try to launch the application from my file manager there is no error popup, and the program does not launch. When I launch the program from the console however I do get this error:

Invalid name "IrScrutinizer-2.4.0-x86_64.appimage", quitting

This error happens even when I rename the file, I even tried to do a appimage extract with the same message occuring.

My system specs are:

OS: Arch Linux x86_64 
Kernel: 6.4.15-x64v2-xanmod1-1 
Shell: bash 5.1.16 
DE: Plasma 5.27.7 
WM: kwin 
CPU: AMD Ryzen 7 1700 (16) @ 3.000GHz 
GPU: AMD ATI Radeon RX 470/480/570/570X/580/580X/590 
Memory: 9717MiB / 32023MiB

**Comment by bengtmartensson** [comment, marked as answer]
It should be `IrScrutinizer-2.4.0-x86_64.AppImage` (with that exact capitalization), not `IrScrutinizer-2.4.0-x86_64.appimage`.

Yes, this is not very user friendly ;-).

**Reply by bengtmartensson** [reply]
> It should be IrScrutinizer-2.4.0-x86_64.AppImage (with that exact capitalization), not IrScrutinizer-2.4.0-x86_64.appimage.

This has been fixed in 4f38a129e2b6f0b1ae7311a7a088b6e2dfb02196

**Comment by bengtmartensson** [comment]
@thyTwilightGoth : did it solve your problem?

**Reply by thyTwilightGoth** [reply]
Yep. Looks like my browser was screwing me over and I did not realize it. TY!
